APIManager.js

```diff
@@ -0,0 +1,67 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+
+    getUsers(){
+        const mainUser = {}
+        const friends = {friends: []}
+        $.get("https://randomuser.me/api/?results=7", function(result){
+            let users = result.results
+            mainUser.firstName = users[0].name.first
+            mainUser.lastName = users[0].name.last
+            mainUser.picture = users[0].picture.large
+            mainUser.city = users[0].location.city
+            mainUser.state = users[0].location.state
+            users.splice(0,1)
+            for(let user of users){
+                friends.friends.push({firstName: user.name.first, lastName: user.name.last})
+            }
+        })
+        this.data.mainUser = mainUser
+        this.data.friends = friends
+    }
+
+    getKanyeQuote(){
+        const quote = {}
+        $.get("https://api.kanye.rest", function(result){
+            quote.text = result.quote
+        })
+        this.data.quote = quote
+    }
+
+    
+    getPokemon(){
+        const pokemon = {}
+        const id = Math.floor(Math.random() * 948) + 1
+        $.ajax({
+            method: "GET",
+            url: `https://pokeapi.co/api/v2/pokemon/${id}/`,
+            success: function(result){
+                pokemon.name = result.name
+                pokemon.image = result.sprites.back_default
+            }
+        })
+        this.data.pokemon = pokemon
+    }
+
+    getAboutMe(){
+        const aboutMe = {}
```

Comment:
also `aboutMe` doesn't have to be an object - use object if you gonna store more than one information about an entity